cognee/modules/storage/utils/__init__.py

```
@@ -0,0 +1,46 @@
+
+class JSONEncoder(json.JSONEncoder):
+    def default(self, obj):
+        if isinstance(obj, datetime):
+            return obj.isoformat()  # Convert datetime to ISO 8601 string
+        elif isinstance(obj, UUID):
+            # if the obj is uuid, we simply return the value of uuid
+            return str(obj)
+        return json.JSONEncoder.default(self, obj)
+
+
+from pydantic import create_model
```

Ruff

Module level import not at top of file

(E402)